pkNX.Structures/GameUtil.cs

```diff
@@ -91,6 +95,7 @@ public static GameVersion GetVersion(int generation)
                 5 => W2,
                 6 => AS,
                 7 => UM,
+                8 => SWSH,
```

Comment:
Should be SH or PLA, not the combined version.

Most of this class is unused (including this method), as it is an old copy from PKHeX's implementation.

Reply:
Got it, will update it 👍 🙂